src/Umbraco.Web/Scheduling/ScheduledPublishing.cs

```diff
@@ -58,9 +55,8 @@ public override bool PerformRun()
                 // run
                 // fixme context & events during scheduled publishing?
                 // in v7 we create an UmbracoContext and an HttpContext, and cache instructions
-                // are batched, and we have to explicitely flush them, how is it going to work here?
-                var publisher = new ScheduledPublisher(_contentService, _logger, _userService);
-                var count = publisher.CheckPendingAndProcess();
+                // are batched, and we have to explicitly flush them, how is it going to work here?
```

Comment:
@zpqrtbnk this comment still exists here but I think we can remove it? This is the comment in v7 about the http context https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Web/Scheduling/ScheduledPublishing.cs#L64 AFAIK we don't need to batch anything, that existed in v7 mostly for legacy load balancing. Now, this will just emit events one at a time and there will be no HttpContext so that means instructions will just be written as they occur right?

Reply:
batching still works with database LB, we group instructions in one database row, to reduce the number of rows we have to read/write - with the current code (everything from v7 removed) each event will trigger an instruction row, I believe, which is not that efficient - looking into it

Reply:
@zpqrtbnk I don't think it's inefficient actually, we now store an instruction count on each row so we know how many rows to load at once